README.md

```diff
@@ -63,11 +60,8 @@ Feel free to check the [issues page](../../issues/).
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- The images and information in this project is taken from [TVMaze](https://www.tvmaze.com/api).
+- We got inspired by the microverse template for this project.
 
 ## 📝 License
-
 This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL] I can see you added `MIT.md` file, and you're using the Microverse template Great job 👌💯 but you forget to customize it, so kindly add your name and customize it 👌😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

![image](https://user-images.githubusercontent.com/45231276/183261501-da1dda92-1813-4057-aaf3-084eb53aa4be.png)